### ISSUE TITLE
How to change AndroidTv Name, Logo, Launcher Logo

### ISSUE THREAD

**Issue by saimirg** [flexn-io/create#172]
Hi, where do change/assign application name, logo, launcher, etc for AndroidTv and FireTv APPs ?

**Comment by saimirg** [comment]
Im using renative.

**Comment by saimirg** [comment]
Im following below folder structure but images under /appConfigs/helloworld/asstes are not being transferred to apps under platformBuilds folders. 

<img width="858" alt="image" src="https://github.com/flexn-io/create/assets/11594998/2adafbd4-79ad-40f6-8359-891b293f30fd">

**Comment by pauliusguzas** [comment]
@saimirg things you described are editable in `appConfigs/[APP_ID]` under renative.json you can change app name, app id, used engines and so on. App icons are also under appConfigs, from example of create-template-starter https://github.com/flexn-io/create/tree/main/packages/template-starter/appConfigs/base/assets/runtime I see that icons are under runtime folder. Can read a bit more about project configuration here if that helps https://renative.org/docs/guides/advanced-configuration

**Comment by saimirg** [comment]
hi @pauliusguzas i've bee doing several tests but the two files ic_log and ic_launcher get overwritten everytime. 
I;ve also tried adding new files on respective folders as documentation says, but again they don't get copied into platformBuilds folder.

**Comment by pauliusguzas** [comment]
@saimirg what versions of rnv and create are you using? And node

**Comment by saimirg** [comment]
Hi, these are the details. I need to understand where in the folder structure i need to place ic_logo and ic_launcher, or even other image names, so that that they are copied to /platformBuilds/... respective app.
 
<img width="563" alt="image" src="https://github.com/flexn-io/create/assets/11594998/6ca2b5fc-47f3-4fa8-b459-24b6b23fbebd">

**Comment by aurimasmi** [comment]
@saimirg it's default folder structure where you should place your assets. As you can see there are multiple drawable folders difference is resolution. Do you see these folders in your file structure? 
![CleanShot2023-10-02at09 53 48](https://github.com/flexn-io/create/assets/16107073/56efcdcc-3977-41aa-a6f7-dcb3a42af8fc)

**Comment by saimirg** [comment]
No, i dont see those folders, i only have base->assets. Shall i create those ?

**Comment by pauliusguzas** [comment]
The issue is that create template uses imported assets. Go to `appConfigs/app/renative.json` and add `"assetSources": null ` under common to not import assets and then you should be able to use your custom assets
